### ISSUE TITLE
nk3: Add PIN overview

### ISSUE THREAD

**Issue by robin-nitrokey** [Nitrokey/pynitrokey#467]
Add a `pin-status` (?) subcommand to the `nk3` command that lists the PIN information for all apps (PIN set: yes/no, remaining retries).  For secrets and FIDO2, this should be easy using existing functionality.  For OpenPGP, we would probably have to introduce a new PIN status query command to the CTAPHID interface.

**Comment by Ingo-Albrecht** [comment]
Such a PIN status overview would be good to have. Perhaps simply add it to `nitropy nk3 status` instead.

Is it possible to obtain a yes/no status via `nitropy nk3 get-config `?

**Comment by robin-nitrokey** [comment]
Yes, we can also consider adding to `status` directly (or with a `--pins` flag).

> Is it possible to obtain a yes/no status via `nitropy nk3 get-config `?

No, the config mechanism does not have access to the PIN status.
